modules/lazyload/module.php

```diff
@@ -87,6 +87,23 @@ private function apply_dominant_color_background( $control, $value, $selector )
 		return $control;
 	}
 
+	private function is_document_support_lazyload( $post_id = null ) {
+		if ( $post_id ) {
+			$document = \Elementor\Plugin::$instance->documents->get( $post_id );
+		} else {
+			$document = \Elementor\Plugin::$instance->documents->get_current();
+		}
```

Comment:
Don't run external logic from a simple function. Always provide a post id.

```suggestion
	private function is_document_support_lazyload( $post_id ) {
		$document = \Elementor\Plugin::$instance->documents->get( $post_id );
```

Reply:
Done

Reply:
Always provide a post id.

Reply:
Sorry, added back.